packages/zoe/src/zoe.js

```diff
@@ -230,43 +234,63 @@ const makeZoe = (additionalEndowments = {}) => {
      * other information, such as the terms used in the instance.
      * @param  {object} installationHandle - the unique handle for the
      * installation
+     * @param  {object} roles - an object mapping roleName key to
+     * issuer values
      * @param  {object} terms - arguments to the contract. These
-     * arguments depend on the contract, apart from the `issuers`
-     * property, which is required.
+     * arguments depend on the contract.
      */
-    makeInstance: (installationHandle, userProvidedTerms) => {
+    makeInstance: (installationHandle, roles, terms = harden({})) => {
```

Comment:
Great move, breaking `roles` out from `terms` and having `terms` default empty.

Reply:
woohoo!